# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/aboutmemory/tests/test_memoryReporters.xul
# Commit: 9fccb38453a4
# Full Hash: 9fccb38453a4fc43b21784c850d4236fdea7d489
# Author: Gabriele Svelto <gsvelto@mozilla.com>
# Date: 2019-11-02 21:51:29
# Regressor Bug: 1586236
# File Overlap Count: 1
# Description:
#   Bug 1586236 - Use memory resource notifications to detect low memory scenarios on Windows; r=dmajor
#   
#   This patch uses the low memory resource notification facility to detect
#   scenarios where physical memory is running low without polling. This is a
#   significant change compared to the previous behavior which measured both
# ==============================================================================

diff -r d60821c527f2 -r 9fccb38453a4 toolkit/components/aboutmemory/tests/test_memoryReporters.xul
--- a/toolkit/components/aboutmemory/tests/test_memoryReporters.xul	Sat Nov 02 08:19:46 2019 +0000
+++ b/toolkit/components/aboutmemory/tests/test_memoryReporters.xul	Fri Nov 01 23:08:59 2019 +0000
@@ -160,8 +160,6 @@
     "JSMainRuntimeRealmsUser",
     "imagesContentUsedUncompressed",
     "storageSQLite",
-    "lowMemoryEventsVirtual",
-    "lowMemoryEventsCommitSpace",
     "lowMemoryEventsPhysical",
     "ghostWindows",
     "pageFaultsHard",